types/monday-sdk-js/index.d.ts

```diff
@@ -0,0 +1,221 @@
+// Type definitions for monday-sdk-js 0.1
+// Project: https://github.com/mondaycom/monday-sdk-js#readme
+// Definitions by: Josh Parnham <https://github.com/josh->
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace monday;
```

Comment:
This should be `mondaySdk` as per https://www.npmjs.com/package/monday-sdk-js#as-a-script-tag-directly-in-your-html-code